blockstore/apps/bundles/apps.py

```diff
@@ -5,4 +5,5 @@
 
 class BundlesConfig(AppConfig):
     name = 'blockstore.apps.bundles'
+    label = 'blockstore_apps_bundles'
```

Comment:
@kdmccormick @arbrandes I do not recall why I made this change but it caused a change in the table names that were being queried (hence the 404s). And which is why it wasn't caught when testing with a fresh devstack+CI run.

Fix coming up.

Reply:
@kdmccormick:

Django uses the app label to define the actual database table name it uses. Devstack didn't capture this because it runs migrations by default during provisioning, although it's easy to see it in the devstack if you (a.) create a library, (b.) deploy the new code, (c.) read the library. It fails with a bundle not found.

In any case, next steps from here:

1. **Keep `bundles` as the default name.** Might impact the maintainability of the platform since it "taints" the app namespace with a very common and unidentifiable name called "bundles", where operators and maintainers will potentially get confused.
2. **Explicitly see the table name in the bundles' application model to "blockstore_bundles".** We keep using the label, which helps with the application's namespace, but still affects the database names.
3. **Ship migrations to rename the database tables.** This is the preferred option in my opinion. I need to investigate further what's the best approach here.

But, while I take a look at (3.), do you have any feedback to share on these? Wanna make sure we are on the same page this time and avoid the bad deployment on the next patch.

Reply:
(Oh, almost forgot, I think we should label the application "blockstore_bundles" rather than "blockstore_app_bundles" since the "app" is redundant.)

Reply:
@jvdm Renaming to `blockstore_bundles` sounds good (I do not recall if this was the reason why I updated the label or if there was something else too) but how about we keep the table names the same for now if possible and then update them when moving them inside edxapp?

Reply:
@symbolist:

> how about we keep the table names the same for now if possible and then update them when moving them inside edxapp?

Is that Option 3, or do you mean something different?

Reply:
I am following up on this on this PR: https://github.com/edx/blockstore/pull/140

The idea is to merge that one, integrate the changes here, at finally tackle the remaining items in the description with a cleaner patch.

CC @symbolist @kdmccormick

Reply:
@symbolist I'm working on rebasing this PR with latest changes in `master`, and can't work out from the discussion here what is meant to happen with the application and table names.

https://github.com/openedx/blockstore/pull/140 [kept `bundles` as the migration app name](https://github.com/open-craft/blockstore/blob/3f4d9792f643abba460237092e3be9e3f28e4d57/blockstore/apps/bundles/migrations/0003_update_character_set.py#L9), but your branch started here [changed the application label to `blockstore_app_bundles`](https://github.com/openedx/blockstore/pull/97/files#diff-320beece25c4a473dca31916ca0238abe9006387f75d4aaa69bb735b8501da6dR12). The discussion above also [mentions `blockstore_bundles`](https://github.com/openedx/blockstore/pull/97/files#r749287941).

I understand that at some point, we're going to have to copy the original data from the blockstore database into the edxapp database. But for where this PR is in the migration, what should the app label be?

Reply:
@pomegranited The table names need to remain the same for now since there is data in the system. I had changed the app labels to add namespacing but if it is not needed for other reasons we can leave it as is for now and consider it when moving the tables into the edxapp db.